examples/existing-instance/outputs.tf

```diff
@@ -0,0 +1,13 @@
+########################################################################################################################
+# Outputs
+########################################################################################################################
+
+output "name" {
+  description = "The name of the watsonx Assistant instance"
+  value       = module.existing_watsonx_assistant_instance.name
+}
+
+output "guid" {
+  description = "GUID of the watsonx Assistant instance"
+  value       = module.existing_watsonx_assistant_instance.guid
+}
```

Comment:
suggest to add the other outputs here too that the module supports

Reply:
Perhaps the existing resource test should actually explicitly check for the outputs too? Just to ensure they are getting returned with values. You could take an approach like this: https://github.com/terraform-ibm-modules/terraform-ibm-cos/blob/a71b4c94888e3a3ed6f0093f1814c64ab5da1631/tests/pr_test.go#L124C2-L124C142

Reply:
Made the change.